tests/results/TextPositionTest.py

```diff
@@ -30,3 +40,58 @@ def test_properties(self):
         uut = TextPosition(8, 39)
         self.assertEqual(uut.line, 8)
         self.assertEqual(uut.column, 39)
+
+
+class TextPositionLineColTest(unittest.TestCase):
+
+    def test_from_position(self):
```

Comment:
I agree with @Uran198 here. The fact that they are specific is the reason why they should be separated.